frontend/src/components/module-viewer/content/blocks/MatchBlock.tsx

```diff
@@ -125,7 +125,7 @@ const MatchBlock = ({
   // Randomize the order of the answers on load
   useEffect(() => {
     setAnswers(answers.sort(() => Math.random() - 0.5));
-  }, []);
+  });

```

Comment:
Removing this array makes this run every time it renders. Is that what we want? Instead, I think this is a case where we can disable the warning.